### PR TITLE
Code Quality UI Updates

### PR DESCRIPTION
**Pull request by MichaelChinn**
use chartsjs to display graph data
iterates over metrics response from /jazz/codeq
removes report-issue to separate component
begins smacss styling

### CODE REVIEW

File: jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss [hunk 1]
```diff
@@ -0,0 +1,176 @@
...
+  text-transform: none;
+  text-underline: none;
+}
+
+.feedback-message {
+  display: flex;
+  flex-direction: column;
+  margin-top: calc(200rem/24)
+}
+
```
Comment (svidhani) [inline jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss]: extra lines?

File: jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss [hunk 2]
```diff
@@ -0,0 +1,176 @@
...
+    align-self: right;
+    /deep/ button.btn-primary{
+      line-height:3rem;
+    }
+  }
+}
+/deep/ button.btn-primary:focus, button.btn-primary:active{
+  background-color: #ed008c !important;
+  color: #fff !important;
+  border: 1px solid #ed008c !important;
```
Comment (svidhani) [inline jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss]: are you guys sticking to using !important?

File: jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss (hunk 1 above)
Comment (MichaelChinn) [reply jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss]: addressed

File: jazz-ui/src/app/core/services/request.service.ts [hunk 3]
```diff
@@ -1,170 +1,171 @@
...
+      .catch((error: any) => {
+        return this.handleError(error, router);
+      })
+
+  }
+
+  private handleError(error: any, router: any) {
+    console.log(error);
+    if (error.status === 401 || error.status === 403) {
+      if (router) {
```
[the remark this answers is not in the document]
Comment (MichaelChinn) [reply jazz-ui/src/app/core/services/request.service.ts]: addressed

File: jazz-ui/src/app/core/services/request.service.ts [hunk 4]
```diff
@@ -1,170 +1,171 @@
...
+        }
+      })
+      .catch((error: any) => {
+        return this.handleError(error, router);
+      })
+
+  }
+
+  private handleError(error: any, router: any) {
+    console.log(error);
```
[the remark this answers is not in the document]
Comment (MichaelChinn) [reply jazz-ui/src/app/core/services/request.service.ts]: addressed

File: jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss (hunk 2 above)
Comment (MichaelChinn) [reply jazz-ui/src/app/primary-components/report-issue/report-issue.component.scss]: this is part of the original component styles

**Pull request closed without merging**